src/models/City.tsx

```diff
@@ -0,0 +1,14 @@
+export default class City {
+
+    id: number;
+    name: string;
+    coord: { lat: number, lon: number};
```

Comment:
Let's create a type for this

Reply:
Do you think that does make sense to create a specific file for this type ?